Adyen.docc/Installation.md

```diff
@@ -40,7 +40,7 @@ pod 'Adyen/SwiftUI'       // SwiftUI apps specific module.
 You can add all modules or select individual modules to add to your integration. But make sure to include each module dependency modules.
 
 * `AdyenDropIn`: DropInComponent.
-* `AdyenSession`: For the new Sessions integration.
+* `AdyenSession`: For the new, simplified checkout flow.
```

Comment:
Maybe just: `automated checkout flow`. Also should start from lower-case

Reply:
We call it simplified checkout, but yeah maybe ommit "new"

Reply:
```
* `AdyenSesson`: For the new, simplified checkout flow.
* `AdyenCard`: the card components.
* `AdyenComponents`: all other payment components except WeChat Pay.
* `AdyenActions`:  action components.
```

To align with the rest of items in this list: `For the new, simplified checkout flow.` -> `simplified checkout flow.`
Maybe also put it above DropIn ? 🤔

Reply:
How about `handler for the simplified checkout flow`?